timesketch/lib/analyzers/manager_test.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for analysis manager."""
+
+from __future__ import unicode_literals
+
+from timesketch.lib.testlib import BaseTest
+from timesketch.lib.analyzers import manager
+
+
+class MockAnalyzer(object):
+    """Mock analyzer class,"""
+    NAME = 'MockAnalyzer'
+
+
+class TestAnalysisManager(BaseTest):
+    """Tests for the functionality of the manager module."""
+
+    manager.AnalysisManager.register_analyzer(MockAnalyzer)
```

Comment:
To mimic how the manager is working this is the closest we get. We don't want to add the mock class for every test method run.

Reply:
I was mistaken, the method I was thinking of is actually `setUpClass()`, and this should run just once per class.  I guess this is OK as is though since these are calling class methods anyway.